src/gui/windows/debug.zig

```diff
@@ -48,11 +50,17 @@ fn flawedRender() !void {
 		y += 8;
 		// TODO: packet loss
 		// TODO: Protocol statistics(maybe?)
+                try draw.print("Connections number: {}", .{main.game.world.?.manager.connections.items.len},  0, y, 8, .left);
```

Comment:
This value will always be 1, since a client will only be connected to one server.
But it would actually be interesting to see the number of clients connected.